modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -16,6 +16,28 @@ New features are added to the language continuously, and occasionally, some feat
 This section lists all of the features that have been removed, deprecated, added, or extended in different Cypher versions.
 Replacement syntax for deprecated and removed features are also indicated.
 
+[[cypher-deprecations-additions-removals-5.21]]
+== Neo4j 5.21
+
+=== New features
+
+[cols="2", options="header"]
+|===
+| Feature
+| Details
+a|
+label:syntax[]
```

Comment:
Nitpick: I think the 'syntax' label makes sense here but char_length and character_length which I compared to does not have that

Reply:
I don't think we need the syntax label here, or in the example below.